libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1383,6 +1383,24 @@ AP_GPS_UBLOX::_parse_gps(void)
                 state.have_gps_yaw = false;
                 state.have_gps_yaw_accuracy = false;
             }
+
+            //report rtk baseline distances from the rover
+            uint32_t nedvalid_mask = static_cast<uint32_t>(RELPOSNED::relPosValid) |
+                                     static_cast<uint32_t>(RELPOSNED::gnssFixOK);
+            if( _buffer.relposned.flags & nedvalid_mask )
+            {
+              // relposN is in cm, relposHPN is in 10*mm
+              state.rtk_baseline_x_mm = (_buffer.relposned.relPosN + _buffer.relposned.relPosHPN*0.01)*10;
+              state.rtk_baseline_y_mm = (_buffer.relposned.relPosE + _buffer.relposned.relPosHPE*0.01)*10;
+              state.rtk_baseline_z_mm = (_buffer.relposned.relPosD + _buffer.relposned.relPosHPD*0.01)*10;
+
+              // also add other status info available
+              state.rtk_baseline_coords_type = 1; // 0: ECEF, 1: NED
+              // 3D accuracy as eucledean norm, in mm.
+              state.rtk_accuracy = sqrt( _buffer.relposned.accN/10 * _buffer.relposned.accN/10 +
```

Comment:
please multiply by 0.1f instead of dividing by ten.

Reply:
Done (not pushed yet). There are a few other divisions by 10 in this file (not my additions), I can fix those in the same commit if needed.

Reply:
`L1221:        state.location.alt    = _buffer.posllh.altitude_msl / 10;` for instance.

Reply:
Sorry, I was mistaken here, those are `uint32` types on the assignment. Fixed and pushed my corrections, @amilcarlucas